examples/getting_started/README.md

```diff
@@ -0,0 +1,607 @@
+
+Now, we have some work to do on our workstation.  First, let's set up
+an inspec directory within our test directories
+
+```
+  $ mkdir -p test/integration/default/controls
+```
+
+This will be our default group of tests.  Now we need to provide a yml
+file with the name of that group within the group directory.  Go ahead
+and create this:
+
+```
+  $ vim test/integration/default/inspec.yml
+```
+
+And add this content
+
+test/integration/default/inspec.yml
+```
+---
+name: default
+```
+
+Save and close the file.
+
+Now, just one more bit of housekeeping.  Our .kitchen.yml is expecting
+an output of our test kitchen instances' hostnames within the output
+variable public_dns. In order to have a hostname within that public_dns
+variable, we need to create an EC2 instance.
+
+First, open up the main config file.
+
+```
+  $ vim main.tf
+```
+
+And add this content:
+
+main.tf
+```
+provider "aws" {
+  access_key = "${var.access_key}"
+  secret_key = "${var.secret_key}"
+  region = "${var.region}"
+}
+
+resource "aws_instance" "example" {
+  ami           = "${var.ami}"
+  instance_type = "${var.instance_type}"
+  key_name      = "${var.key_name}"
+}
+```
+
+Save and close the file.  Notice that we used some variable values
+there?  We need to add these into our variables.tf file.
+
+Go ahead and create it.
+
+```
+  $ vim variables.tf
+```
+
+And add in this content:
+
+variables.tf
+```
+variable "access_key" {}
+variable "secret_key" {}
+variable "key_name" {}
+variable "region" {}
+variable "ami" {}
+variable "instance_type" {}
+```
+
+And, finally, we need to add in some values for these variables.
+
+Open up your testing.tfvars file
+
+```
+  $ vim testing.tfvars
+```
+
+And add in this content (substitute in the appropriate values for your
+AWS account, region, etc.)
+
+testing.tfvars
+```
+access_key = "my_aws_access_key"
+secret_key = "my_aws_secret_key"
+key_name = "my_aws_key_pair_name"
+region = "us-east-1"
+ami = "ami-fce3c696"
+instance_type = "t2.medium"
+```
+
+Save and close the file.
+
+Finally, let's define that output that will be expected by our
+.kitchen.yml.
+
+Go ahead and open up your output.tf file.
+
+```
+  $ vim output.tf
+```
+
+And add this content
+
+```
+output "public_dns" {
+  value = "${aws_instance.example.public_dns}"
+}
+```
+
+Ok!  Now we're ready to finally create some test kitchen instances!
+
+Go ahead and run
+```
+  $ bundle exec kitchen converge
+```
+
+NOTE: If you receive the error "Error launching source instance:
+VPCResourceNotSpecified: The specified instance type can only be
+used in a VPC", check out [this documentation for help correcting it](http://docs.rightscale.com/faq/EC2_t2.x_Instance_Type_Requirement.html)
+
+And it should converge successfully and you should see output that
+includes
+
+```
+Outputs:
+
+  public_dns = your_aws_instance_public_ip
+```
+
+Let's first try running the tests as is - even though we haven't written
+any yet.
+
+```
+  $ bundle exec kitchen verify
```

Comment:
This command is assuming the default security group has SSH open to your incoming public IP address. Probably want to note that or show them how to add it.